users/views.py

```diff
@@ -164,4 +171,279 @@ def get(self, request, user_id, token):
             return Response(
                 {'status': 'error', 'message': 'An error occurred during verification.'},
                 status=status.HTTP_500_INTERNAL_SERVER_ERROR
-            )
\ No newline at end of file
+            )
+
+
+User = get_user_model()
+def get_default_role():
+    return UserRole.objects.get(role="user")
```

Comment:
Add error handling for missing "user" roles (e.g., DoesNotExist exception).
Consider caching the result if this function is called frequently.